package.json

```diff
@@ -20,27 +20,27 @@
     "css-loader": "^6.7.3",
     "elasticlunr": "^0.9.5",
     "find-cache-dir": "^3.3.2",
-    "gatsby": "^4.24.8",
-    "gatsby-plugin-catch-links": "^4.24.0",
-    "gatsby-plugin-google-gtag": "^4.24.0",
-    "gatsby-plugin-less": "^6.24.0",
-    "gatsby-plugin-manifest": "^4.24.0",
-    "gatsby-plugin-offline": "^5.24.0",
-    "gatsby-plugin-postcss": "^5.24.0",
-    "gatsby-plugin-react-helmet": "^5.24.0",
-    "gatsby-plugin-sharp": "^4.24.0",
-    "gatsby-plugin-typography": "^4.24.0",
-    "gatsby-remark-autolink-headers": "^5.24.0",
-    "gatsby-remark-copy-linked-files": "^5.24.0",
+    "gatsby": "^4.25.4",
```

Comment:
#201

Reply:
Could you share the info? I cannot understand what this comment means.

Reply:
Hello @hong6316 . sorry for the misunderstanding. When the PR was created, #201 was still active and I wanted to mention that I am updating gatsby version to latest v4 version in this PR and we should wait for 201 to be merged before merging this PR. In the meantime, 201 was merged and my comment is obsolete

Reply:
:+1: